### PR TITLE
MathMystics (Feature/components)

### PR DESCRIPTION
**Pull request by lucy-sees**
In this part of the project I implemented the following tasks:

- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called components.
- Inside components, I created a new Calculator.js file.
- In Calculator.js, I created a React component that matches this design:
calculator

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+class Calc extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
```
Comment (joskalenda) [inline src/components/Calculator.js:8]: - [ ] Nice way of declaring the calc component but it is not what required, this is a class component while you are asked to create the calculator as a functional component. please make the changes for your code too like: ---> 

` function Calculator() {
  return <div> ..... </div>
}
`

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @lucy-sees , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Great desiggn
- [ ].. ✅ Green linters
- [ ].... ✅ Good readme

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (lucy-sees) [reply src/components/Calculator.js:8]: Alright. Thank you for highlighting that

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @lucy-sees , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done updating the app component to a functional component. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**